webserver/views/api/v1/core.py

```diff
@@ -29,14 +29,14 @@ def get_low_level(mbid):
     an offset as a query parameter in the form of ?n=x, where x is an integer
     starting from 0
     """
-    mbid, offset = _validate_data_arguments(mbid, request.args.get("n"))
+    offset = _validate_offset(request.args.get("n"))
     try:
-        return jsonify(db.data.load_low_level(mbid, offset))
+        return jsonify(db.data.load_low_level(str(mbid), offset))
     except NoDataFoundException:
         raise webserver.exceptions.APINotFound("Not found")
 
 
-@api_bp.route("/<string:mbid>/high-level", methods=["GET"])
+@bp_core.route("/<uuid:mbid>/high-level", methods=["GET"])
 @crossdomain()
 def get_high_level(mbid):
     """Endpoint for fetching high-level data.
```

Comment:
After reading the documentation website, I'm not sure about this style of writing. Yes, this function "implements an endpoint", but when you read it in the documentation I think it would read better as "Get high level data for a given MusicBrainz Recording id" (or similar) instead. What do you think?